chat.js

```diff
@@ -618,6 +653,15 @@ class CommandContext {
 			}
 
 			if (room) {
+				if (room.r9k && !room.isMuted(user)) {
```

Comment:
I thought so too but I could still trigger the mute over and over again. I'll test again I guess?